controller/userController.js

```diff
@@ -35,29 +35,23 @@ const login = (req, res) => {
 
                 // Generate response
                 const response = {
-                    auth: true,
-                    status: 'Logged in',
                     user: user,
                     token: token
                 };
-
                 return res.status(HttpStatus.OK).json(response);
             } else {
                 return res.status(HttpStatus.FORBIDDEN).json({
-                    status: StatusEnum['ERROR'],
                     message: 'Incorrect email or password'
                 });
             }
         } else {
             return res.status(HttpStatus.NOT_FOUND).json({
-                status: StatusEnum['ERROR'],
                 message: 'Please provide a valid email and password.'
             });
         }
     }).catch(err => {
         return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
-            status: StatusEnum['ERROR'],
-            message: 'Internal server error.'
+            message: ValidationUtil.buildErrorMessage(err, 'login', 'user')
```

Comment:
I am not certain you should be using the `buildErrorMessage` function here in the outer `catch` since I don't see how those errors are user related (user has not entered something invalid that fails mongoose validation, he simply didn't input valid credentials for login).